docker/run_tests.sh

```diff
@@ -14,13 +14,26 @@ geant4_version=10.04
 
 DAGMC_dir=${install_dir}/DAGMC-moab-${moab_version}
 
+# test DAGMC CMake configuration file using LD_LIBRARY_PATH to find DAGMC
+cd ${build_dir}/DAGMC-moab-${moab_version}/src/cmake/test_config
+cmake . -DDAGMC_ROOT=${DAGMC_dir}
+make all test
```

Comment:
I'm not sure how this is working on CI. When I tested it myself, it gave errors unless I put hdf5, moab, and dagmc in my LD_LIBRARY_PATH.

Perhaps we can require LD_LIBRARY_PATH for now and then whenever I get around to submitting a good PR for the RPATH update we can revisit it

Reply:
Yes. An early morning oversight on my part. 

For now, I'll add this initial test before we add the DAGMC install dir to the `LD_LIBRARY_PATH` but yeah setting the RPATH would be ideal at some point.